lib/App/DuckPAN/Web.pm

```diff
@@ -276,6 +278,29 @@ sub request {
 		my @calls_goodie;
 		my @ids;
 
+		my $repo = $app->get_ia_type;
+
+		##########
+		# FATHEAD
+		if ($repo->{name} eq "Fathead") {
+
+			my $output_txt = path( $app->fathead_output() );
+			if ($output_txt->exists) {
```

Comment:
Why not bail earlier?

If `fathead_output` and doesn't exist, we don't want to go through the effort of setting anything up. We should just kill the server immediately as there's nothing useful we can do.

We may wish to throw an informative error if they are in a non-FatHead repo and have specified `output_txt`.

(`"output_txt" is only valid for FatHead Instant Answers"`)